code/game/jobs/job/science.dm

```diff
@@ -68,6 +68,7 @@
 	spawn_positions = 5
 	supervisors = "the Chief Research Overseer"
 	difficulty = "Medium."
+	department_account_access = TRUE
```

Comment:
```suggestion
	department_account_access = FALSE
```
Middle-gound is that robotics get access as they are MEANT to be a implanter wielding and dealing person selling and buying mech gear/selling stuff